view/rooms/app.js

```diff
@@ -829,6 +837,34 @@ function closeConfirmationModal() {
   );
 }
 
+/**
+ * Room not found so we display error message
+ * @param {DOM} document document to display not found modal on
+ */
+function displayNotFoundModal(doc) {
```

Comment:
Is there a reason why we're using both "" and `` to define strings in this function instead of ''?